src/store/modules/Overview/OverviewStore.js

```diff
@@ -0,0 +1,42 @@
+import api from '../../api';
+
+const OverviewStore = {
+  namespaced: true,
+  state: {
+    serverInfo: null,
+    serverModel: '--',
+    serverManufacturer: '--',
+    serverSerialNumber: '--'
+  },
+  getters: {
+    serverInfo: state => state.serverInfo,
+    serverModel: state => state.serverModel,
+    serverManufacturer: state => state.serverManufacturer,
+    serverSerialNumber: state => state.serverSerialNumber
+  },
+  mutations: {
+    setServerInfo: (state, serverInfo) => (state.serverInfo = serverInfo),
+    setServerModel: (state, serverModel) => (state.serverModel = serverModel),
+    setServerManufacturer: (state, serverManufacturer) =>
+      (state.serverManufacturer = serverManufacturer),
+    setServerSerialNumber: (state, serverSerialNumber) =>
+      (state.serverSerialNumber = serverSerialNumber)
+  },
+  actions: {
+    getServerInfo({ commit }) {
+      api
+        .get('/xyz/openbmc_project/inventory/system')
+        .then(response => {
+          const serverInfo = response.data.data;
+          commit('setServerModel', serverInfo.Model);
+          commit('setServerManufacturer', serverInfo.Manufacturer);
```

Comment:
It looks like this is being returned as an empty string. In that case, our `--` is never shown and the name is empty. If we want to use --, we will need to check for a value and only commit only if it exists.

```
if (serverInfo.Manufacturer) {
  commit(...)
}